types/array_basic.go

```diff
@@ -15,13 +14,18 @@ const BasicArraySizeCache = 100000
 var fastSumHashKey = toBytes32([]byte("hash_fast_sum64_key"))
 
 type basicArraySSZ struct {
-	hashCache *ccache.Cache
+	hashCache *ristretto.Cache
 	lock      sync.Mutex
 }
 
 func newBasicArraySSZ() *basicArraySSZ {
+	cache, _ := ristretto.NewCache(&ristretto.Config{
+		NumCounters: BasicArraySizeCache, // number of keys to track frequency of (1M).
```

Comment:
What is 1M? 1 million? the value is 100k.
```suggestion
		NumCounters: BasicArraySizeCache, // number of keys to track.
```